camel-openmrs-fhir/src/main/java/org/openmrs/eip/fhir/spring/OpenmrsFhirConfiguration.java

```diff
@@ -51,16 +69,7 @@ public void beforeApplicationStart(CamelContext camelContext) {
 				FhirConfiguration fhirConfiguration = new FhirConfiguration();
 				FhirContext ctx = FhirContext.forR4();
```

Comment:
Aren't you supposed to get rid of these lines 69 and 70 since you're are now using the new spring bean?

Reply:
No, still need both. The new spring bean `IGenericClient` should be set to `FhirConfiguration` which is then set camel FHIR component.